p4proto/p4src/linux_networking/README_LINUX_NETWORKING_WITH_ECMP.md

```diff
@@ -219,13 +216,55 @@ Note: Port number used in ovs-p4ctl commands are target datapath indexes(unique
   ```
 
 #### 14. Configure ECMP for underlay connectivity
-  - **Rule for: To reach link partner multiple paths are configured, packets can be hashed to any port where the nexthop's ARP is learnt**
-  ```
-    ip route add 40.1.1.2 nexthop via 50.1.1.2 dev TAP1 weight 1 nexthop via 60.1.1.2 dev TAP2 weight 1
-  ```
+- **Rule for: To reach link partner multiple paths are configured, packets can be hashed to any port where the nexthop's ARP is learnt**
```

Comment:
Add some notes on the packet fields used for next hop selection when multiple routes are available.